src-backbone/app/js/models/conditionals/conditionalNode.js

```diff
@@ -298,6 +298,16 @@ const ConditionalNode = Backbone.Model.extend({
     // Template helpers
     //--------------------------------------------------------------------------
 
+    getElementQuestion: function() {
+        if (!(this.isCriteriaNode() && this.get('criteria_element') > 0)) {
```

Comment:
Changing to `(!this.isCriteriaNode() || !(this.get('criteria_element') > 0))` triggers a JSHint warning  `Confusing use of '!'.` It cannot just compare the value <= 0 because test fails when the value is undefined and will fall through the if block